lib/platform/gitlab/index.ts

```diff
@@ -440,12 +440,35 @@ export async function getPrList(): Promise<Pr[]> {
   return config.prList;
 }
 
+async function ignoreApprovals(pr: number): Promise<void> {
+  try {
+    const url = `projects/${config.repository}/merge_requests/${pr}/approval_rules`;
+    const { body: rules } = await gitlabApi.getJson<{ name: string }[]>(url);
+    const ruleName = 'renovateIgnoreApprovals';
+    const zeroApproversRule = rules?.find(({ name }) => name === ruleName);
+    if (!zeroApproversRule) {
+      await gitlabApi.postJson(url, {
+        body: {
+          name: ruleName,
+          approvals_required: 0,
+        },
+      });
+    }
+  } catch (err) {
+    logger.warn({ err }, 'GitLab: Error adding approval rule');
+  }
+}
+
 async function tryPrAutomerge(
   pr: number,
   platformOptions: PlatformPrOptions
 ): Promise<void> {
   if (platformOptions?.gitLabAutomerge) {
     try {
+      if (platformOptions?.gitLabIgnoreApprovals) {
+        await ignoreApprovals(pr);
+      }
```

Comment:
Does it really need `gitLabAutomerge`

Reply:
I think it makes sense to only removal approvals if it's an automerging MR

Reply:
Ok. Makes sense.